test/e2e/flowaggregator_test.go

```diff
@@ -177,19 +178,21 @@ func TestFlowAggregator(t *testing.T) {
 
 	if v4Enabled {
 		t.Run("IPv4", func(t *testing.T) { testHelper(t, data, podAIPs, podBIPs, podCIPs, podDIPs, podEIPs, false) })
+		t.Run("testFlowAggregatorAntctl", func(t *testing.T) { testFlowAggregatorAntctl(t, data, podAIPs, podBIPs, false) })
```

Comment:
Please use different test name for IPv4 and IPv6 runs, or moving `testFlowAggregatorAntctl` inside to `testHelper`

Reply:
Thanks Yongming for reviewing. Moved the test to `testHelper`.